esti/auth_test.go

```diff
@@ -66,134 +54,134 @@ func TestAdminPolicies(t *testing.T) {
 	require.Equal(t, http.StatusNoContent, resDeleteUser.StatusCode(), "Admin unexpectedly failed to delete the user")
 }
 
-// Test Super User Policies: AuthManageOwnCredentials, FSFullAccess, RepoManagementReadAll
-func TestSuperUserPolicies(t *testing.T) {
+// Test Super Permissions: AuthManageOwnCredentials, FSFullAccess, RepoManagementReadAll
+func TestSuperPermissions(t *testing.T) {
 	ctx, logger, repo := setupTest(t)
-	gid := "SuperUsers"
+	gid := "Supers"
 
-	// generate the SuperUser client
-	superUserClient := newClientFromGroup(t, ctx, logger, gid)
+	// generate the Super client
+	superClient := newClientFromGroup(t, ctx, logger, gid)
```

Comment:
nit: no value in `gid` IMO just call `superClient := newClientFromGroup(t, ctx, logger, "Supers")`
Same for the following cases

Reply:
Done!